src/assets/_project/_blocks/layout/section-nav/_section-nav.scss

```diff
@@ -46,10 +46,14 @@
       margin: 0;
       padding: 1em;
       color: $qg-dark-grey;
-      &.active{
+      &.active {
         color: #212529;
         font-weight: 700;
       }
+      // If a link gets active, focus, or focus-within event apply the blue outline
+      &:active, &:focus, &:focus-visible {
```

Comment:
This is already included in the `qg-button-outline-decoration` mixin, so why repeat it here?